cointop/coins_table.go

```diff
@@ -136,6 +136,10 @@ func (ct *Cointop) GetCoinsTable() *table.Table {
 					})
 			case "24h_volume":
 				text := humanize.Monetaryf(coin.Volume24H, 0)
+				if ct.State.scaleNumbers {
+					numScale, numSuffix := humanize.Scale(coin.Volume24H)
+					text = humanize.Numericf(numScale, 1) + numSuffix
```

Comment:
would it make sense to use a single function that combines these two methods?

Reply:
already in my local :)